.github/workflows/release.yml

```diff
@@ -38,6 +38,8 @@ jobs:
   publish-release:
     needs: [build-release, build-spdx]
     runs-on: ubuntu-24.04
+    permissions:
+      contents: write
```

Comment:
It's strange how this was never needed before. Maybe it has something to do with a recent transition to enterprise GH or something along those lines.

Reply:
@danieldegrasse - did you test this in a CI workflow in a fork?

Reply:
> Yes I did- here is a job failing based on the v18.4.0-rc1 tag: https://github.com/danieldegrasse/tt-zephyr-platforms/actions/runs/15338904861/job/43162145000. I created a 18.4.0-rc2 tag on my fork with the commit in that PR, and the release action succeeds there: https://github.com/danieldegrasse/tt-zephyr-platforms/actions/runs/15339297827

Should mention that in the PR description too. Thanks for testing.